crates/transaction-generator-lib/src/publishing/publish_util.rs

```diff
@@ -116,7 +118,7 @@ impl PackageHandler {
 #[derive(Clone, Debug)]
 pub enum Package {
     Simple(
-        Vec<(String, CompiledModule)>,
+        Vec<(String, CompiledModule, u32)>,
```

Comment:
Can we please add a `///` above saying that the `String` is the module name and `u32` is the binary format version?